src/test/java/uk/gov/hmcts/sptribs/exception/DocumentUploadOrDeleteExceptionTest.java

```diff
@@ -82,8 +83,6 @@ void testUpdateDocumentUploadOrDeleteException() throws Exception {
             documentManagementController.deleteDocument(CASE_TEST_AUTHORIZATION, documentInfo.getDocumentId());
```

Comment:
As per the method, uploadDocument controller needs to be called instead of deleteDocument.